wsi_deid/web_client/views/ItemView.js

```diff
@@ -226,21 +281,21 @@ wrap(ItemView, 'render', function (render) {
                 return;
             }
             elem.find('.g-hui-redact').remove();
+            if (hideField(keyname, hideFieldPatterns)) {
+                elem.closest('tr').css('display', 'none');
+            }
```

Comment:
We should make the `if (showControls)` an `else if (showControls)`, as it does no good to show a control on a hidden field.  This could be done in a separate PR, as it was neither caused by nor directly related to this one.

Reply:
I just created #220 to track this issue.